src/senaite/core/scripts/_reindex.py

```diff
@@ -65,4 +65,4 @@ def run(app):
 
 
 if __name__ == "__main__":
-    run(app)  # noqa
+    run(app) # noqa: F821
```

Comment:
`app` seems to be a global variable. is there a better way to handle this? for example

```suggestion
    app = globals()['app']
    run(app)
```